packages/grafana-ui/src/types/graph.ts

```diff
@@ -0,0 +1,9 @@
+import { DisplayValue } from './displayValue';
+
+/** View model projection of a series */
+export interface GraphSeriesVM {
+  label: string;
+  color: string;
+  data: any[][]; // [x,y][]
```

Comment:
any reason for any?

Reply:
matches the series data model.. but maybe we should use:  `number | null` and force a conversion/check?

I like either GraphPlot or SeriesPlot.  Lean towards `GraphPlot` since it is options for <Graph/> and i thin the file `graph.ts` makes more sense than `plot.ts`